com.github.tchx84.Flatseal.json

```diff
@@ -44,8 +45,8 @@
             "sources": [
                 {
                     "type": "archive",
-                    "url": "https://download.gnome.org/sources/libhandy/1.0/libhandy-1.0.2.tar.xz",
-                    "sha256": "3ad78d0594165c7e8150f662506d386552825e693aa3679744af96bd94dc1c2d"
+                    "url": "https://download.gnome.org/sources/libhandy/1.2/libhandy-1.2.1.tar.xz",
```

Comment:
libhandy is available in the sdk, you can remove it

Reply:
hey @Lctrs , 

mind sending an upstream PR?

Reply:
Done